MapleServer2/Types/Player.cs

```diff
@@ -36,46 +35,47 @@ public class Player
         public Levels Levels { get; private set; }
         public int MapId;
         public int TitleId;
-        public List<int> Titles = new List<int> { 0 };
-        public List<short> Insignias = new List<short> { 0 };
+        public List<int> Titles;
```

Comment:
I don't feel that there is an issue with having default values set along with the variable declaration. I see no reason to set them separately within the constructor. 

On another note, I feel that there is value in changing these variables into properties, instead of using public fields. It probably won't change much for now, but can be helpful for setting default values or for doing validation on values you try to set a variable to. This is a suggestion and not a requested change.

Reply:
Since we we have fields being instantiated in the constructor already, I moved all instantiation into the constructor for consistency. As for them being properties, there isn't any use for them at the moment but if there was I wouldn't be against making them properties instead of fields.

Reply:
I still can't say I'm a fan of setting them their, but it doesn't really hurt anything in the end. It is more consistent I suppose, though I really don't feel like they belong there. Its fine for now I suppose, someone else can weigh in if they'd like, but it works fine and is consistent at the very least.

Reply:
The two that were there originally were necessary because they pass an instance of the player, the others don't so it's technically not necessary, and I moved them purely for consistency. But if it's better practice to leave them separated, then I am okay with reverting the change.

Reply:
I would say that it is better practice to leave them separated in this case. In the case for the ones that were necessary, those need to stay since those are set by values passed in through the constructor. The other variables don't really need to be set within the constructor, so they can just have a default variable set when you declare them.